data/categories.txt

```diff
@@ -474,6 +474,7 @@ fa:خود پرداز
 en:2RV Facilities|4Caravan|Vanlife|5Campervan|5Motorhome
 be:Для аўтадамоў|5Аўтадом|5Трэйлер|5Караван|Дом на колах
 ru:Для автодомов|5Автодом|5Трейлер|Дом на колёсах
+tr:4Karavan|5Karavan Tesisleri
```

Comment:
Here you don't need the second synonym, because the first Karavan will appear as a hint for the user after typing the first 4 letters "kara".
```suggestion
tr:4Karavan Tesisleri
```

After changing it here, it also should be renamed in strings.txt to work.

Reply:
Please check it now on your device to confirm, that 💰 for example is a working synonym for money.

Reply:
I already renamed it in strings.txt -- now I'm editing categories.txt